### ISSUE TITLE
Internal error - Build status for the function's pkg

### ISSUE THREAD

**Issue by JohnMicl** [fission/fission#974]
### What happened:
When I followed [official  docs](https://docs.fission.io/0.12.0/workflows/) to start my first workflow, there is an internal error, it must be something wrong with this [fortunewhale.wf.yaml](https://raw.githubusercontent.com/fission/fission-workflows/0.6.0/examples/whales/fortunewhale.wf.yaml).

It is obviously different from fortunewhale.wf.yaml file described in the docs
### What you expected to happen
If you follow steps to create your first workflow in [docs](https://docs.fission.io/0.12.0/workflows/), you could get an internal error , like this
`Error creating service for function: Internal error - Build status for the function's pkg : fortunewhale-wf-yaml-vkci.default is : failed, can't fetch deployment`

if you modify the file fortunewhale.wf.yaml like this

```
apiVersion: 1
output: WhaleWithFortune
tasks:
  InternalFuncShowoff:
    run: noop
  GenerateFortune:
    run: fortune
    requires:
    - InternalFuncShowoff
  WhaleWithFortune:
    run: whalesay
    inputs:
      body: "{ output('GenerateFortune') }"
    requires:
    - GenerateFortune
```
It can work! 
### What I want
I hope it can be fixed. It took me some time to fix this problem

**Comment by erwinvaneyk** [comment]
Hi @JohnMicl -Thanks for the thorough report! I have tried to reproduce your issue by following the instructions on the page you linked to (https://docs.fission.io/0.12.0/workflows/). However, I did not encounter the issue you are describing.

Looking at the error, I think it might be an error related to Fission's build system, since the error does not indicate an issue with the workflow itself (but rather parts of Fission not being ready yet).

After the workflow you changed worked, did you try the original one again? This would verify whether the error is transient or not 🙂
